### ISSUE TITLE
`read` doesn't complete with a relatively simple SVG with ellipse

### ISSUE THREAD

**Issue by abey79** [abey79/vpype#77]
File exhibiting the issue: [input-2.svg.zip](https://github.com/abey79/vpype/files/5418771/input-2.svg.zip)

**Comment by zxsq-cc** [comment]
Did some messing with this in inkscape. [this version](https://github.com/zxsq-cc/scratchpad/blob/master/input-2a.svg) doesnt show the large ellipses in inkscape but *does* in `vpype [...] view` 
[this one](https://github.com/zxsq-cc/scratchpad/blob/master/input-2.svg) has the objects on a layer. (was trying to recreate the 2a file that worked)

I believe it is an issue either with the size (the looks are enourmous, coordinates around 1million) or them being in root layer not a numbered layer

**Comment by jimmymorris** [comment]
Found some more info out. It seemingly has issues with the `transform` attribute. If I run commands against the below file it hangs.

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="700" height="1000">
    <ellipse cx="525" cy="750" rx="75" ry="75" fill="rgb(0, 0, 0)" stroke="none"></ellipse>
    <ellipse cx="478.3309524416879" cy="703.3309524416878" rx="66" ry="66"
      transform="rotate(-315 478.3309524416879 703.3309524416878)" stroke="rgb(0, 0, 0)"></ellipse>
</svg>
```

If I remove the `transform` then the read completes and is quick.

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="700" height="1000">
    <ellipse cx="525" cy="750" rx="75" ry="75" fill="rgb(0, 0, 0)" stroke="none"></ellipse>
    <ellipse cx="478.3309524416879" cy="703.3309524416878" rx="66" ry="66" stroke="rgb(0, 0, 0)"></ellipse>
</svg>
```

**Comment by abey79** [comment]
The issue with this SVG is that the ellipse are physically HUUUUGE, so the conversion to polyline yields an enormous amount of segment. #88 will significantly improve the situation, but care is still required.

One way is to crop to the dimension (read doesn't do it yet although it probably should #94):
```
vpype -v read -m  my_work/users/input-2.svg crop 0 0 700 1000 show
```

Another way is to simplify upon read. With #88 this is not implicit anymore and `-s` must be used. #88 also introduces an option for multiprocessed SVG conversion, which should be used here:

```
vpype -v read -m -sp my_work/users/input-2.svg show
```

This still takes 1m40s on my computer to execute.

**Comment by abey79** [comment]
Finally, I've addressed #94 in #88, so the geometries are cropped by default. This file now reads fine with:
```
vpype -v read my_work/users/input-2.svg show
```

11s on my computer

<img width="752" alt="image" src="https://user-images.githubusercontent.com/49431240/100158545-a1e00d80-2eac-11eb-869e-537f01efc713.png">
